mappings/net/minecraft/client/gui/widget/factory/LayoutWidgetWithTitledRowFactories.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/unmapped/C_vjsalahs net/minecraft/client/gui/widget/factory/LayoutWidgetWithTitledRowFactories
```

Comment:
I don't love this name. I think that `LayoutWidgetFactories` would be perfectly acceptable, since it's specified in the method names that you're creating a widget with a titled row. It's redundant to say it twice, and I think the method name is a much better place to put it than the class name.

(do note I can't use github selections here, since to properly rename a class it has to happen from enigma. otherwise the mapping file won't be renamed)

Reply:
I agree, that's totally reasonable.  I'll do that.